readmes/unity_viz.md

```diff
@@ -0,0 +1,171 @@
+
+### Add Global Costmap Visualization
+
+- Although the default visualization suite is provided as a prefab, you are free to make changes to the suite for your own use. Still on the `OccupancyGrid` object's Inspector window, click `Add Component`. Begin searching for `Occupancy Grid Default Visualizer` and add it to the object. You should now have two occupancy grid visualizers on this object!
+
+    > The `+` icon on the newly added component indicates that it is added to a prefab, but the changes on it exist in this particular scene, and not the prefab file itself.
+
+- On the newly added component, specify this `Topic` as `/global_costmap/costmap`, and similarly, update the `Offset` to something that will be below the `/map` offset, e.g. `0.01`.
+
+- To set the default material to the new Occupancy Grid visualizer, assign the `Material` field to the OccupancyGrid material, found in `Packages/Robotics Visualization/Runtime/Materials/OccupancyGrid`.
+
+    > You can find this file by searching in the Project window. Note that you will have to change the Search type from "In Assets" to "All". 
+
+    ![](images/viz_search.png)
+
+- You may now want to differentiate between the two occupancy grid visualizers--in the `Unoccupied` Cell Color field of these two visualizers, feel free to start changing colors! We kept the `/map` as white, and changed the `/global_costmap/costmap` to be a light blue.
+
+    ![](images/viz_occupancygrids.png)
+
+- Once again, enter Play mode. 
+
+    > You may have to restart your ROS nodes before re-entering Play mode if you encounter unexpected issues. Check the [Troubleshooting](#troubleshooting) section below.
+
+    In the HUD's Topics window, toggle on the `3D` for `/global_costmap/costmap` now.
+
+You should now see the two maps updating in realtime! As you send goal poses to Turtlebot, the occupancy grid drawings should now update as expected.
+
+![](images/viz_maps.png)
+
+## Laser Scan Visualization
+
+Finally, let's visit how the laser scan sensor is being visualized in the scene. Using the Visualization Package, point cloud-type visualizations are highly customizable. This section will walk through customization options for a sensor_msgs/LaserScan visualization for your Nav2 project.
+
+- If you are still in Play mode, exit it. 
+
+- In the scene Hierarchy, once again expand the `DefaultVisualizationSuite`. This time, expand the `sensor_msgs` object and select `LaserScan` to open its Inspector.
+
+    In the `Topic` field, enter `/scan`.
+
+- For messages with stamped headers, there is an option to customize the coordinate frame tracking per visualization. This is set via the `TF Tracking Settings`; click it to expand the options. 
+    
+    To create each drawing as children of respective *frame_id* GameObjects, change the `Type` to `Track Latest`. The `TF Topic` should be left as the default `/tf`.
```

Comment:
Suggestion: "In this case, we want the laser scan to be drawn with respect to the most recent transform we have for its parent, the turtlebot base_link [check that this is right], so change..."